Modules/Devices/src/Devices.Application/Identities/Commands/ApproveDeletionProcess/Handler.cs

```diff
@@ -41,6 +46,8 @@ public async Task<ApproveDeletionProcessResponse> Handle(ApproveDeletionProcessC
 
         _eventBus.Publish(new TierOfIdentityChangedDomainEvent(identity, oldTierId, newTierId));
 
+        await _notificationSender.SendNotification(identity.Address, new DeletionProcessApprovedNotification(deletionProcess.GracePeriodEndsAt?.DaysUntilDate() ?? 0), cancellationToken);
```

Comment:
`GracePeriodEndsAt` is never null after the deletion process is approved. So you might as well use null suppression operator. Or, if you check, you should at least throw an exception instead of passing 0.